src/components/Post.jsx

```diff
@@ -45,8 +46,10 @@ function Post(prop) {
   const [comment, setComment] = useState("");
   const [likesNo, setLikesNo] = useState(likecount ? likecount.length : 0);
   const [anchorEl, setAnchorEl] = useState(null);
+  const[editCaption,setEditCaption]=useState(caption)
```

Comment:
please maintain a proper format, try using auto formatter like prettier to format the code :)